### DISCUSSION TITLE
Bigger preview?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft-search/pnp-modern-search#3007] [Ideas]
Hi!
Is there anyway to make the previews bigger when clicking on the thumbnails? I tried to change the code under edit template in resultspart but nothing happends. Anyone got an idea for this?

![Preview](https://user-images.githubusercontent.com/6117320/104463672-5abff000-55b2-11eb-88ec-3a2529f9b490.PNG)

**Comment by FranckyC** [comment]
Hi @socker22 , not possible currently but could an enhancement in the future, However, this will be low priority as you could understand ;)

**Reply by buildbod** [reply]
It's been a couple years since the original idea was raised. It would be good to improve the preview and especially being able to define basic parameters like width. At the moment [there is a hack](https://github.com/microsoft-search/pnp-modern-search/issues/1794#issuecomment-1430177757) that involves identifying the host container and styling it e.g.



```
.calloutContentContainer_329bf4c0 {
                                    width: 1000px;
                                }

```
